packages/adapters/wagmi/src/client.ts

```diff
@@ -726,6 +726,7 @@ export class WagmiAdapter implements ChainAdapter {
           ])
           if (connector) {
             this.syncConnectedWalletInfo(connector)
+            ProviderUtil.setProvider(this.chainNamespace, await connector.getProvider())
```

Comment:
We might need to call `ProviderUtil.setProviderId` as well?

Reply:
@KannuSingh

Reply:
@enesozturk  what would be the correct way to get `providerId` of connected provider?

Reply:
Got it, its `connector.id`